neuro_data/static_images/data_schemas.py

```diff
@@ -964,76 +892,19 @@ class Member(dj.Part):
         name                    : varchar(50) unique # string description to be used for training
         """
 
-    _template = 'group{group_id:03d}-{animal_id}-{session}-{scan_idx}-{preproc_id}'
+    def fill(self):
```

Comment:
I think the fill should probably be a class mathod (i.e., not receive the self) just `def fill:`, and it will need to be called StaticMultiDataset.fill() rather than StaticMultiDataset().fill(), no need to instantiate the object if it won't be used

Reply:
Yep fair point? Rushed through this and missed that.